packages/cursorless-engine/src/actions/FollowLink.ts

```diff
@@ -24,11 +29,12 @@ export default class FollowLink implements SimpleAction {
       .openLink(target.contentRange);
 
     if (!openedLink) {
-      await this.actions.executeCommand.run(
-        targets,
-        "editor.action.revealDefinition",
-        { restoreSelection: false },
-      );
+      let commandId = this.openInSplit
+        ? "editor.action.revealDefinitionAside"
+        : "editor.action.revealDefinition";
+      await this.actions.executeCommand.run(targets, commandId, {
+        restoreSelection: !this.openInSplit,
+      });
```

Comment:
Actually I think we should just move this whole `if` statement to somewhere in https://github.com/cursorless-dev/cursorless/blob/main/packages/cursorless-vscode/src/ide/vscode/VscodeOpenLink.ts? Feels pretty vscode-specific

WDYT @AndreasArvidsson

Reply:
Totally agree

Reply:
@jaresty I think makes sense to do that in this PR as it's a small change and you'll be touching that code already

Reply:
Sounds good. I'll take a look soon.

Reply:
I'm looking at this but I'm having a hard time understanding what the change needs to be. It seems that the VSCode follow link method currently only handles URIs. I'm not quite sure how to make the change without doing a significant restructure here, which makes me think that I'm misunderstanding something about the intention or the request. Can you give me a little bit more specific details about what you're hoping to see change? I also don't mind if you want to make the change yourself (in case it's easier to do it than to explain).

Reply:
I think I figured it out. One thing that I'm still unsure about is that in the previous version I needed to pass the targets to the execute action, but now it doesn't seem to be necessary. The functionality does seem to work however.